### DISCUSSION TITLE
Sending path for the specific page from Gatsby to Strapi

### DISCUSSION THREAD

**Discussion by MotaZor94** [strapi/strapi#8622] [Legacy Topics]
Hello everyone,

We are currently running approx 70-100 pages live from Strapi, which is connected to Gatsby atm.

Does anyone know if there is a simple way of connecting the PATH of the page to some field in Strapi so the user can see the URL of the generated page when he creates it?

Would it be a simple POST request to Strapi with the data that you want to store in the field?

Cheers guys and enjoy your week!

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question, we are currently planning to decomission our GitHub Discussions in early 2021 and we don't want to lose the great questions asked by our community.

Questions should be directed to [our forum](https://forum.strapi.io/). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).
